### PR TITLE
Update dual-decision credit file definition

### PR DESCRIPTION
**Pull request by dara123**

### CODE REVIEW

**Review by adamaltman — DISMISSED** [review]
Edited:  One minor text suggestion to accept please.

File: openapi/components/schemas/KycDocument/CreditFileMatches.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ properties:
       - other
     x-enumDescriptions:
       single-source: The credit file can be used as a single source of proof for identity decisions.
-      dual-source: The credit file can be used as a factor in identity decisions but needs accompanying data such as a government issued ID.
+      dual-source: The credit file can be used as a source of proof where the credit bureau agency found two reliable sources on their end.
```
Comment (adamaltman) [inline openapi/components/schemas/KycDocument/CreditFileMatches.yaml:34]:
```suggestion
      dual-source: The credit file can be used as a source of proof where the credit bureau agency found two reliable trade sources.
```

**Comment by dara123** [comment]
@Rebilly/php please review

**Comment by dara123** [comment]
@Rebilly/docs please review

**Review by vykhrystiuk — APPROVED** [review]

**Comment by dara123** [comment]
https://github.com/orgs/Rebilly/teams/php please review

**Comment by dara123** [comment]
@Rebilly/php please review

**Review by mikerebilly — APPROVED** [review]

**Pull request merged**